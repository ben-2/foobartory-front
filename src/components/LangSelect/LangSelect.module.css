.title {
  font-size: 70px;
  font-weight: bold;
  font-style: italic;
  font-family: monospace;
}