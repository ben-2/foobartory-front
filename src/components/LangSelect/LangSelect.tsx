import styles from './LangSelect.module.css';

export function LangSelect() {
  return (
    <>
      <div className={styles.title}>
        Selecteur de langue
      </div>
    </>
  );
}
