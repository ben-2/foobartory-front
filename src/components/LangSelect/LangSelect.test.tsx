import { LangSelect } from './LangSelect';
import { shallow } from 'enzyme';

describe('<LangSelect />', () => {
  it('renders lang select list', () => {
    const wrapper = shallow(<LangSelect />)
    expect(wrapper).toMatchSnapshot();
  })
});
