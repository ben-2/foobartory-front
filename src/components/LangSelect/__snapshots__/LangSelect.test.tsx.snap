// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LangSelect /> renders lang select list 1`] = `ShallowWrapper {}`;
